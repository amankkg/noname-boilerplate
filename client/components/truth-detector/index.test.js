import React from 'react'

import Component from './index'

test('should just pass', () => {
  expect(<Component value={0} />).toMatchSnapshot()
})

test('should return `Aye!` if value is 42', () => {
  expect(<Component value={42} />).toMatchSnapshot()
})

test('should return `Naah!` if value is not 42', () => {
  expect(<Component value={41} />).toMatchSnapshot()
})
