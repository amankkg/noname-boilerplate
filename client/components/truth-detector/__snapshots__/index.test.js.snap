// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should just pass 1`] = `
<TruthDetector
  value={0}
/>
`;

exports[`should return \`Aye!\` if value is 42 1`] = `
<TruthDetector
  value={42}
/>
`;

exports[`should return \`Naah!\` if value is not 42 1`] = `
<TruthDetector
  value={41}
/>
`;
